media/memory-demo/wasm.k

```diff
@@ -5,7 +5,7 @@ module WASM
 
```

Comment:
This file probably shouldn't be changed, it should be saved in the state that @hjorthjort gave the presentation in I think.